public/js/clients/chrome/InspectorBackend.js

```diff
@@ -0,0 +1,704 @@
+/**
```

Comment:
Where did these files in the `chrome` folder come from? Might be worth documenting (unless they already are and I missed that)